continuum/scenarios/permutations.py

```diff
@@ -0,0 +1,72 @@
+from typing import Callable, List
+
+import torch
+
+from continuum.datasets import _ContinuumDataset
+from continuum.scenarios import TransformationIncremental
+
+from torchvision import transforms
+
+
+class Permutations(TransformationIncremental):
+    """Continual Loader, generating datasets for the consecutive tasks.
+
+    Scenario: Permutations scenarios, use same data for all task but with pixels permuted.
+    Each task get a specific permutation, such as all tasks are different.
+
+    :param cl_dataset: A continual dataset.
+    :param nb_tasks: number of tasks in the continuum.
+    :param base_transformations: List of transformation to apply to all tasks.
+    :param seed: initialization seed for the permutations.
+    """
+
+    def __init__(
+            self,
+            cl_dataset: _ContinuumDataset,
+            nb_tasks: int,
+            base_transformations: List[Callable] = None,
+            seed=0
```

Comment:
Let the users define all tasks permutations seeds, thus a `list` instead of an `int` with an argument `seeds` (no need to add prefix "list" as in the [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)).

And it would mean we don't need `nb_tasks`.

And in this case `nb_tasks` would be equal to `len(seeds) + 1` because the first task has no permutations. But I'm willing to discuss about this last point.